src/components/DatePicker/DatePicker.tsx

```diff
@@ -43,6 +43,8 @@ export interface DatePickerProps {
    * @default 0
    */
   weekStartsOn?: number;
+  /** Visually hidden prefix text for selected days on single selection date pickers */
+  dayAccessibilityLabelPrefix?: string;
```

Comment:
Should this be optional? If it makes the experience better for screen readers it should be required.

Reply:
It won't always be needed. When you have a single date picker, you won't need it. When you have a ranged date picker we have defaults (added in this pull request). But when you're creating a custom ranged date picker (see order list date filters) we need custom prefixes.
<img width="358" alt="Screen Shot 2021-01-05 at 10 18 25 AM" src="https://user-images.githubusercontent.com/24610840/103663732-6cb0eb80-4f3f-11eb-89a0-80db8c827764.png">